packages/styles/src/button/button.spec.js

```diff
@@ -0,0 +1,32 @@
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
+const exquisite = require('exquisite-sst');
+const colors = require('colors');
+
+// Wait 2000 ms after the loaded event to take the screenshot
+const delay = 1000
+
+// Reference image
+const input = path.resolve(__dirname, 'img/reference.png');
```

Comment:
how do we take the reference screenshot? With Exquisite a first time?

Reply:
yep